templates/hotgraphic.hbs

```diff
@@ -1,7 +1,7 @@
 {{! Maintainers - Kevin Corry}}
 <div class="hotgraphic-inner component-inner" role="region" aria-label="{{_globals._components._hotgraphic.ariaRegion}}" {{#if _globals._components._hotgraphic.ariaRegion}}tabindex="0"{{/if}}>
   {{> component this}}
-  <div class="hotgraphic-widget component-widget {{#if _useGraphicsAsPins}}tile{{else}}pin{{/if}}">
+  <div class="hotgraphic-widget component-widget {{#if _useGraphicsAsPins}}tile{{else if  _useNumbersAsPins}}numbers{{else}}pin{{/if}}">
```

Comment:
Will need to bump the required framework version in `bower.json` to `2.0.11` if we're using the `{{else if}}` syntax – this was added in Handlebars v3.